docs/content/preview/yugabyte-platform/anywhere-automation/anywhere-cli.md

```diff
@@ -0,0 +1,269 @@
+---
+title: YugabyteDB Anywhere CLI
+headerTitle: YugabyteDB Anywhere CLI
+linkTitle: CLI
+description: Use YugabyteDB Anywhere CLI to create and manage resources in YBA.
+headcontent: Install the CLI, configure default settings, and set up autocompletion
+techPreview: /preview/releases/versioning/#feature-availability
+menu:
+  preview_yugabyte-platform:
+    parent: anywhere-automation
+    identifier: anywhere-cli
+    weight: 50
+type: docs
+rightNav:
+  hideH4: true
+---
+
+The [YugabyteDB Anywhere (YBA) Command Line Interface](https://github.com/yugabyte/yugabyte-db/blob/master/managed/yba-cli/docs/yba.md) (CLI) is an open source tool that enables you to interact with YBA using commands from your shell. With minimal configuration, you can start running commands from the command line that implement functionality equivalent to that provided by the browser-based YBA interface.
+
+The CLI can only be used with YugabyteDB Anywhere v2024.1 or later.
+
+The CLI is in [Tech Preview](/preview/releases/versioning/#feature-availability). Commands and options may change before GA.
+
+## Install CLI
+
+Your YugabyteDB Anywhere installation includes the CLI binary, located in `/opt/yugabyte/yugaware/yba-cli`. The following versions are available:
+
+- yba_cli-<ver>-darwin-amd64
+- yba_cli-<ver>8-darwin-arm64
+- yba_cli-<ver>-linux-amd64
+- yba_cli-<ver>-linux-arm64
```

Comment:
Looks like md format does not like <ver> and so it does not show up in the final doc. Can you please adjust this?